Bento/Renderable/NativeView.swift

```diff
@@ -0,0 +1,17 @@
+import UIKit
+
+public typealias NativeView = UIView
+
+extension NativeView {
+    @nonobjc static func generate() -> UIView {
+        if let viewType = self as? (UIView & NibLoadable).Type {
+            return viewType.loadFromNib()
+        }
+
+        return self.init()
+    }
+
+    @nonobjc static var typeName: String {
+        return String(describing: self)
```

Comment:
We can also append the value of `ObjectIdentifier` from the meta type which is going to be constant in a particular invocation.

Reply:
But in any case:

> fully relying on `String(describing: self)` is dangerous

This fortunately doesn't apply to Bento, since its implementation actually caters for type mismatches so as to support a few boundary cases. The only con is that it would cost you a bit more time spent in view instantiation. :)

Reply:
> The only con is that it would cost us a bit more time spent in view instantiation. :)

I think view instantiation every time will be costy for scrolling performance.
What do you think?

Reply:
Yeah. It could potentially be an issue. I have switched to use `String.init(reflecting:)`, and added tests to catch any change in `String.init(reflecting:)` behaviour.